python/ray/autoscaler/v2/scheduler.py

```diff
@@ -175,7 +187,32 @@ def try_schedule(
                 - the utilization score for this node with respect to the current
                 resource requests being scheduled.
         """
-        pass
+        # Track the resource requests that cannot be scheduled on this node.
+        unschedulable_requests = []
+
+        def _sort_resource_request(req: ResourceRequest) -> Tuple:
+            """
+            Sort the resource requests by:
```

Comment:
why this particular sort order?

Reply:
We had the similar logic in the existing implementation (other than having placement constraints):

https://github.com/ray-project/ray/blob/c21f7a209a0ccbadd3bdf73eb3bd6240fabc629c/python/ray/autoscaler/_private/resource_demand_scheduler.py#L902-L914